kpops/components/base_components/kafka_connect.py

```diff
@@ -1,26 +1,73 @@
+from __future__ import annotations
+
 import json
-import os
+import logging
 from abc import ABC
+from functools import cached_property
 from typing import NoReturn
 
+from pydantic import Field
 from typing_extensions import override
 
-from kpops.cli.pipeline_config import ENV_PREFIX
-from kpops.component_handlers.kafka_connect.connector_handler import KafkaConnectorType
-from kpops.component_handlers.kafka_connect.model import KafkaConnectConfig
+from kpops.component_handlers.helm_wrapper.helm import Helm
+from kpops.component_handlers.helm_wrapper.helm_diff import HelmDiff
+from kpops.component_handlers.helm_wrapper.model import (
+    HelmRepoConfig,
+    HelmUpgradeInstallFlags,
+)
+from kpops.component_handlers.helm_wrapper.utils import trim_release_name
+from kpops.component_handlers.kafka_connect.model import (
+    KafkaConnectConfig,
+    KafkaConnectorType,
+    KafkaConnectResetterConfig,
+    KafkaConnectResetterValues,
+)
 from kpops.components.base_components.base_defaults_component import deduplicate
 from kpops.components.base_components.models.from_section import FromTopic
 from kpops.components.base_components.pipeline_component import PipelineComponent
+from kpops.utils.colorify import magentaify
+
+log = logging.getLogger("KafkaConnector")
 
 
 class KafkaConnector(PipelineComponent, ABC):
     _type = "kafka-connect"
     app: KafkaConnectConfig
 
+    repo_config: HelmRepoConfig = HelmRepoConfig(
+        repository_name="bakdata-kafka-connect-resetter",
+        url="https://bakdata.github.io/kafka-connect-resetter/",
+    )
+    namespace: str
+    version: str = "1.0.4"
+    resetter_values: dict = Field(
```

Comment:
In defaults.yaml this only works in snake case. Most other properties seem to work in camelCase. What needs to be done here @disrupted ?

Reply:
We should add the config class and inherit the CamelCase class